demos/slinky-slurm-demo.html

```
@@ -0,0 +1,679 @@
+      </svg>
+    </figure>
+  </div>
+</section>
+
+<!-- 4 · RECIPE -->
+<section class="slide" data-title="Recipe">
+  <div class="slide-inner">
+    <div class="label">03 · Generate</div>
+    <h2>Recipe: criteria become a resolved plan</h2>
+    <div class="code">
+      <div class="bar"><span class="dot" style="background:#e5534b"></span><span class="dot" style="background:#e3a008"></span><span class="dot" style="background:#76b900"></span><span class="name">recipe</span><button class="copy">copy</button></div>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)